charts/rancher-monitoring/v0.2.3/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+description: Provides monitoring for Kubernetes which is maintained by Rancher 2.
+engine: gotpl
+maintainers:
+- name: thxCode
+  email: frank@rancher.com
+name: rancher-monitoring
+sources:
+- https://github.com/coreos/prometheus-operator
+version: 0.2.3
+appVersion: 0.2.3
```

Comment:
Can we split the base copy and modification changes into separate commits? I'm assuming all you changed was the version / appVersion and will be modifying the questions.yml based on Caleb's comment in https://github.com/rancher/system-charts/pull/459#discussion_r630383265, but this should be in a separate commit.

The base copy commit should generally be the equivalent of doing `cp -R charts/rancher-monitoring/v0.2.2 charts/rancher-monitoring/v0.2.3`.

Reply:
done.